.github/workflows/main.yaml

```diff
@@ -42,6 +42,8 @@ jobs:
         export LD_LIBRARY_PATH=/opt/ucx/lib:/opt/ucc/lib:$LD_LIBRARY_PATH
         /opt/ucx/bin/ucx_info -e -u t
         export UCX_LOG_LEVEL=info
+        export TORCH_UCC_ENABLE_HEALTH_CHECK=1
+        export TORCH_SHOW_CPP_STACKTRACES=1
```

Comment:
`TORCH_SHOW_CPP_STACKTRACES` is not necessary, but I feel that adding this would be convenient for debugging when there are failures in the CI.

Reply:
My recommendation is to keep the `TORCH_UCC_*` name for now and do the renaming when we upstream.